modules/upgrade-upgrading-cli.adoc

```diff
@@ -0,0 +1,147 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-cluster.adoc
+
+[id="upgrade-upgrading-cli_{context}"]
+= Updating a cluster by using the CLI
+
+If updates are available, you can update your cluster by using the
+OpenShift CLI (`oc`).
+
+You can find information about available {product-title} advisories and updates
+link:https://access.redhat.com/downloads/content/290/ver=3.11/rhel---7/3.11.98/x86_64/product-errata[in the errata section]
+of the Customer Portal.
+
+[IMPORTANT]
+====
+If you installed the OpenShift 4 Beta Drop 5, you can update to {product-title}
+{product-version}.
+====
+
+.Prerequisites
+
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+* Log in to the cluster as user with `cluster-admin` privileges.
+* Install the `jq` package.
+
+.Procedure
+
+. Ensure that your cluster is available:
++
+----
+$ oc get clusterversion
+
+NAME      VERSION     AVAILABLE   PROGRESSING   SINCE     STATUS
+version   4.1.0-rc.4   True        False         24h       Cluster version is 4.1.0-rc.4
+----
++
+If your `VERSION` is `4.1.0-rc.4`, you installed OpenShift 4 Beta Drop 5, and
+you can update to {product-title} {product-version}.
```

Comment:
I got "beta drap 5 should be rc.3" from mail "Announcing OpenShift 4.1 - Beta 5 [4.1.0-rc.3]. please double confirm about the rc-beta relation.

Reply:
You're right. And the last email I see is "Announcing OpenShift 4.1 - Beta 7 [4.1.0-rc.5]." In the replies, Sebastian Jug says that Beta 7 is still using rc.4. That matches the RHCOS bits here: https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.1/

I'm updating the PR for Beta 7 and rc.4.